platform/viewer/src/connectedComponents/Viewer.js

```diff
@@ -182,6 +173,7 @@ class Viewer extends Component {
 
     const measurementApi = new MeasurementApi(timepointApi, {
       onMeasurementsUpdated: this.onMeasurementsUpdated,
+      server: activeServer,
```

Comment:
🤔 I wonder if we'll ever want Measurements to persist despite returning to the study list